content/news/2024-09-05-first-direct-collaboration-nge-osrd.md

```diff
@@ -0,0 +1,24 @@
+---
+title: First direct collaboration of OpenRail projects
+date: 2024-09-05
+image:
+  src: images/news/collaboration-people.jpg
+  alt: Four people holding each others' wrists
+  class: image-position-center
+---
+
+A half year after the official launch of the OpenRail Association, five Open Source projects have already become part of the organization. Even after this short time, two of these projects - one from the SNCF, the other from the SBB - have already started to collaborate directly. This collaboration increases the efficiency of both developments and creates synergies that benefit both projects and, ultimately, their customers. This collaboration was made possible thanks to the OpenRail Association.
```

Comment:
Do we want to talk about "customers" ou "users" ?

Reply:
Ah, the idea here was to mention the benefit for the railway customers, the ones taking trains, delivering cargo, and other transportation clients. Or is that too far-stretched?